cli/azd/cmd/auth_login.go

```diff
@@ -318,15 +318,38 @@ func (la *loginAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			var msg string
 			switch res.Status {
 			case contracts.LoginStatusSuccess:
-				msg = "Logged in to Azure."
+				msg = "Logged in to Azure"
 			case contracts.LoginStatusUnauthenticated:
-				msg = "Not logged in, run `azd auth login` to login to Azure."
+				msg = "Not logged in, run `azd auth login` to login to Azure"
 			default:
 				panic("Unhandled login status")
 			}
 
-			fmt.Fprintln(la.console.Handles().Stdout, msg)
-			return nil, nil
+			// get user account information - login --check-status
+			details, err := la.authManager.LogInDetails(ctx)
+
+			// error getting user account
+			if err != nil {
+				log.Printf("error: getting signed in account: %v", err)
+				fmt.Fprintln(la.console.Handles().Stdout, msg+".")
```

Comment:
Let's get rid of the ending period here for all messages -- it makes copy-pasting much harder